Makefile

```diff
@@ -22,25 +22,21 @@ help: ## Display this help
 lib: $(SOURCES) ## build skipper library
 	go build ./...
 
-.PHONY: bindir
-bindir:
-	mkdir -p bin
```

Comment:
That's not so great because it breaks the build for new contributors

Reply:
No, bin is created by go build automatically